hash/dc.xml

```diff
@@ -5756,34 +6393,6941 @@ http://segaretro.org/
+		<part interface="dc_gdrom" name="gdrom1">
+			<feature name="part_id" value="Di Gi Charat Fantasy"/>
+			<diskarea name="gdrom">
+				<disk name="di gi charat fantasy - shokai genteiban (jpn)" sha1="525b860f800222d26dc53f17f420d93d270c504c"/>
+			</diskarea>
+		</part>
+		<!--
+		<part interface="dc_gdrom" name="gdrom2">
+			<feature name="part_id" value="Soundtrack Arrangement CD"/>
+			<diskarea name="gdrom">
+				<disk name="di gi charat fantasy - shokai genteiban (jpn) (soundtrack arrangement cd)" status="nodump"/>
+			</diskarea>
+		</part>
+		-->
+	</software>
+
+	<software name="digichrt" cloneof="digichrtsg">
+		<!--
+		<rom name="Di Gi Charat Fantasy v1.004 (2001)(Broccoli)(NTSC)(JP)[!].gdi" size="88" md5="d991bd2c09518469d6f5d6b9cd9ab0fe" sha1="d76ab79ab71b25e06992bb7f2f0317c35285a34a"/>
+		<rom name="track01.bin" size="12451488" md5="6250e66cb5cc4e236d8881a641c1fb2d" sha1="36b369c7a5ff67096264aaf5fc8986e3d09184a7"/>
+		<rom name="track02.raw" size="1735776" md5="8659426a29f58924f5f0458cc720098b" sha1="e12b9ae5ebc79b7da5271bce29df450671ce5d93"/>
+		<rom name="track03.bin" size="1185760800" md5="5b860d94ecff2e0a3919edf3f7ffb6a7" sha1="1eefcd921904020a50792c7fb989ce0aad362064"/>
+		-->
+		<description>Di Gi Charat Fantasy (Jpn)</description>
+		<year>2001</year>
+		<publisher>Broccoli</publisher>
+		<info name="serial" value="T-46302M"/>
+		<info name="release" value="20010906" />
+		<info name="alt_title" value="デ･ジ･キャラット ファンタジー" />
+		<part interface="dc_gdrom" name="gdrom">
+			<diskarea name="gdrom">
+				<disk name="di gi charat fantasy (jpn)" sha1="525b860f800222d26dc53f17f420d93d270c504c"/>
+			</diskarea>
+		</part>
+	</software>
+
+	<!--
+	<software name="celsior">
+		<description>Digital Catalog: Celsior (Jpn)</description>
+		<year>2000</year>
+		<publisher>Toyota</publisher>
+		<info name="serial" value="T-39214M"/>
+		<info name="alt_title" value="デジタル カタログ セルシオ" />
+		<part interface="dc_gdrom" name="gdrom">
+			<diskarea name="gdrom">
+				<disk name="digital catalog celsior (jpn)" status="nodump"/>
+			</diskarea>
+		</part>
+	</software>
+	-->
+
+	<software name="trackman">
+		<!--
+		<rom name="My Trackman v1.003 (1999)(Shouei)(NTSC)(JP)[!].gdi" size="150" md5="d1f91f2cdc39aa5a357f908e700c85be" sha1="922be73f9fbe407a6e45c63d7e3e0553da1a58cc"/>
+		<rom name="track01.bin" size="2039184" md5="341a536d93c347603d9b12ac4db9fb01" sha1="8e68abbca0678c03fc1901810ea992ad2fac98e3"/>
+		<rom name="track02.raw" size="1735776" md5="8659426a29f58924f5f0458cc720098b" sha1="e12b9ae5ebc79b7da5271bce29df450671ce5d93"/>
+		<rom name="track03.bin" size="1043271936" md5="fa017848809ed79f32545351d5abb35d" sha1="f01a454325fde984086d8680bb6459b6ff6874f6"/>
+		<rom name="track04.raw" size="1559376" md5="a83898683b5470e32508c8dddb59244d" sha1="21f632a4cd97c58265cb548db3bc015fc95f54c7"/>
+		<rom name="track05.bin" size="140223888" md5="e0ff3b28b831709c91018cafc2f33b6e" sha1="50c3a037c2b8dfb4f775101bacc8da99571845f
```

Comment:
Software List device 'cd_list': dc.xml: Errors parsing software list:
   dc.xml(8160.3): Unknown tag: feature

feature must be before diskarea